### PR TITLE
typo fix

### PR DESCRIPTION
**Pull request by bearice**

### CODE REVIEW

**Comment by aboch** [comment]
LGTM
(CI failure is unrelated)

**Comment by aboch** [comment]
Please sign your commit first and push again.

File: docs/ipam.md [hunk 1]
```diff
@@ -266,4 +266,4 @@ If true, on `CreateEndpoint()` request, libnetwork will generate a random MAC ad
 
 ## Appendix
 
-A Go extension for the IPAM remote API is available at [docker/go-plugins-helpers/ipam](https://github.com/docker/go-plugins-helpers/tree/master/ipam)
\ No newline at end of file
+A Go extension for the IPAM remote API is available at [docker/go-plugins-helpers/ipam](https://github.com/docker/go-plugins-helpers/tree/master/ipam)
```
Comment (chenchun) [inline docs/ipam.md]: What changed on this line?

Comment (bearice) [inline docs/ipam.md]: Missing EOL at end of file (this is added by github web editor)

**Comment by chenchun** [comment]
LGTM

**Pull request merged**